packages/components/src/components/expandable/expandable.stories.ts

```diff
@@ -0,0 +1,143 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/test';
+import { waitUntil } from '@open-wc/testing-helpers';
+
+const { argTypes, parameters } = storybookDefaults('sd-expandable');
+const { overrideArgs } = storybookHelpers('sd-expandable');
+const { generateTemplate } = storybookTemplate('sd-expandable');
+
+export default {
+  title: 'Components/sd-expandable',
+  component: 'sd-expandable',
+  args: overrideArgs([
+    { type: 'slot', name: 'default', value: '<div class="slot slot--border slot--text h-16">Default slot</div>' }
+  ]),
+  argTypes,
+  parameters: { ...parameters }
+};
+
+/**
+ * Expandable shows a brief summary and expands to show additional content.
+ */
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      args
+    });
+  }
+};
+
+/**
+ * Use the inverted attribute to make an expandable with inverted colors.
+ */
+export const Inverted = {
+  parameters: { controls: { exclude: 'inverted' } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: { type: 'attribute', name: 'inverted' }
+      },
+      args,
+      options: {
+        templateBackgrounds: { alternate: 'y', colors: ['rgb(var(--sd-color-primary, 0 53 142))', 'white'] }
+      }
+    });
+  }
+};
+
+/**
+ * Expandable can be used with background options of white, neutral-100 and primary-100. If inverted, can be used with primary. The default slot can be used with 2 variants for alternate expandable experiences: lead text and paragraph.
+ */
+export const Samples = {
```

Comment:
it looks to me like the gradients need to be adjusted according to the background?!?